Day 2/BMI Calculator.py

```diff
@@ -0,0 +1,4 @@
+height = input("enter your height in m: ")
```

Comment:
M = meter ? 
So If I have 180 cm that's => 1.8 or 1,8 , but then you transform float type into int in this place 
`result = int(height)/(float(weight)*(float(weight)))` , and program has been broken.